### PR TITLE
YAG-36-integrate-search-functionality-with-read-one

### PR DESCRIPTION
**Pull request by S4kRiPp4**
Kyle and I came up with this in class today. 

Basically, the search button/text field is fully functional and will display only the row of the productID searched and its attributes, inside the JTable.

We also added a method that clears the Jtable that is called within the search button. A new file Search.java is included where the readOne method lives.

### CODE REVIEW

**Review by Jesusxx-240 — APPROVED** [review]
Great work on this. Everything looks good to me. You had some merging conflicts, but I resolved them so it should be good to merge.

File: Main_Project/src/Views/ManageInventoryGUI.java [hunk 1]
```diff
@@ -291,6 +301,14 @@ public void refreshProducts(){
 
 
     }
+
+    //Method to refresh the productsTable JTable 
+    public void refreshProductsSearch(){
```
Comment (chillyssa) [inline Main_Project/src/Views/ManageInventoryGUI.java]: @S4kRiPp4 I am seeing this method as refreshProductsSearch() still. Per our discord conversation can you updated the method name to clearProductsTable() and then also make the method call adjustment on line 266 ?

**Comment by S4kRiPp4** [comment]
> 

sweet thanks for doing that!

**Review by chillyssa — CHANGES_REQUESTED** [review]

File: Main_Project/src/Views/ManageInventoryGUI.java (hunk 1 above)
Comment (S4kRiPp4) [reply Main_Project/src/Views/ManageInventoryGUI.java]: that's weird it must of not saved

**Review by chillyssa — APPROVED** [review]
Re-reviewed- might have some comment conflicts to address after merge but we will need to fix this after merging

**Pull request merged**